packages/experimental/config.js

```diff
@@ -6,3 +6,5 @@
  */
 
 export default 'Experimental';
+
+export const sectionTitle = 'Legacy/Automation';
```

Comment:
If I recall correctly, per https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/328 we shouldn't need this anymore.

```suggestion
```